teamscale-client/src/main/java/com/teamscale/client/ITeamscaleService.java

```diff
@@ -57,6 +57,7 @@
 			@Path("projectName") String projectName,
 			@Query("format") EReportFormat format,
 			@Query("t") CommitDescriptor commit,
+			@Query("revision") String revision,
```

Comment:
TBH, as this is kind of a central API within the Teamscale ecosystem, I would expect a more elaborate Javadoc of this method, in particular also for this parameter.